assets/src/components/mapPage/vehiclePropertiesCard.tsx

```diff
@@ -200,12 +200,7 @@ const VehicleNearestIntersection = ({
 }
 
 const VehicleLocationStreetViewButton = ({ vehicle }: { vehicle: Vehicle }) => (
```

Comment:
suggestion: maybe `VehicleLocationStreetViewButton` should be renamed or collapsed?

Reply:
Oh heh, thanks for catching this, I definitely meant to rename it.

Reply:
a8d12a49